### ISSUE TITLE
Common Authorization line for all requests

### ISSUE THREAD

**Issue by pkshima** [Huachao/vscode-restclient#555]
This is an awesome extension of productivity !!

But could you please make it possible to specify the security in a single place.
Currently, I have to add an extra line after each request line. For eg.

###
GET {{baseUrl}}/abc/xyz/
Authorization: Bearer {{jwtstring}}

###
GET {{baseUrl}}/abc/xyz/dejkde
Authorization: Bearer {{jwtstring}}

###
GET {{baseUrl}}/abc/xyz/jkdewjfk
Authorization: Bearer {{jwtstring}}

It makes the file unnecessarily long and distracting. Would be so nice to have it write only once near the top of the file.

Thanks in advance !

**Comment by bpeicher** [comment]
This would be very useful.
